fontforge/sfd.c

```diff
@@ -4368,6 +4374,8 @@ Undoes *SFDGetUndo( FILE *sfd, SplineChar *sc,
 
 	    if( !strmatch(tok,"ImageX:"))
 	    {
+	    enum MIME mime = SFDGetImageMIME(sfd);
+	    if (mime == false) exit(1);
```

Comment:
No... don't do this. Hard exits are bad, they're the worst kind to debug.

Plus won't this always evaluate to true, because `PNG` is the only enum value?

Reply:
@jtanx I tested it and I can still write out PNGs. I thought that too but it does work for me. I have no clue why :-)

Reply:
btw `SFDGetImageMIME` prints out errors so it shouldn't be too hard to debug. `sfd.c` does this in a lot of places, IError, then exit. What should I do instead?

Reply:
Yeah, that's not a good enough reason to be honest. Don't overload the type by returning `false` when it's an enum. Add an `UNKNOWN` type into the enum and return that from that function/check for that here.

Reply:
Will do